app/models/exercise/authorship.rb

```diff
@@ -0,0 +1,6 @@
+class Exercise::Authorship < ApplicationRecord
+  belongs_to :exercise
+  belongs_to :author, # rubocop:disable Rails/InverseOf
```

Comment:
Rather than disabling the InverseOf here, you should probably add it. So that `User` has:
```ruby
has_many :authorships, source: :author
has_many :authored_exercises, through: :authorships
```

(I think that's right)

Reply:
I copied it from `Exercise::Prerequisites`:

```ruby
class Exercise::Prerequisite < ApplicationRecord
  belongs_to :exercise
  belongs_to :concept, # rubocop:disable Rails/InverseOf
    class_name: "Track::Concept",
    foreign_key: :track_concept_id
end
```

Does that also need updating?